Compilation failure with matio 1.5.27

In conda-forge matio got update to 1.5.17, and this error appeared:

~~~
[6/36] Building CXX object CMakeFiles/matioCpp.dir/src/File.cpp.o
FAILED: CMakeFiles/matioCpp.dir/src/File.cpp.o 
/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/_build_env/bin/x86_64-conda-linux-gnu-c++ -DmatioCpp_EXPORTS -I/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work/include -I/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work/build/Autogenerated -isystem /home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/include/eigen3 -fvisibility-inlines-hidden -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/include -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work=/usr/local/src/conda/libmatio-cpp-0.2.4 -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place=/usr/local/src/conda-prefix -O3 -DNDEBUG -fPIC -MD -MT CMakeFiles/matioCpp.dir/src/File.cpp.o -MF CMakeFiles/matioCpp.dir/src/File.cpp.o.d -o CMakeFiles/matioCpp.dir/src/File.cpp.o -c /home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work/src/File.cpp
/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work/src/File.cpp: In member function 'std::vector<std::__cxx11::basic_string<char> > matioCpp::File::variableNames() const':
/home/conda/feedstock_root/build_artifacts/matio-cpp-split_1716299022551/work/src/File.cpp:294:33: error: invalid conversion from 'char* const*' to 'char**' [-fpermissive]
  294 |         char** list = Mat_GetDir(m_pimpl->mat_ptr, &list_size);
      |                       ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                 |
      |                                 char* const*
[7/36] Building CXX object CMakeFiles/matioCpp.dir/src/CellArray.cpp.o
[8/36] Building CXX object CMakeFiles/matioCpp.dir/src/Struct.cpp.o
ninja: build stopped: subcommand failed.
~~~

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@
 # BSD-2-Clause license (https://opensource.org/licenses/BSD-2-Clause).
 
 cmake_minimum_required(VERSION 3.10)
-project(matioCpp VERSION 0.2.4 LANGUAGES CXX)
+project(matioCpp VERSION 0.2.5 LANGUAGES CXX)
 
 # Defines the CMAKE_INSTALL_LIBDIR, CMAKE_INSTALL_BINDIR and many other useful macros.
 # See https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

--- a/src/File.cpp
+++ b/src/File.cpp
@@ -291,7 +291,7 @@ std::vector<std::string> matioCpp::File::variableNames() const
     if (isOpen())
     {
         size_t list_size;
-        char** list = Mat_GetDir(m_pimpl->mat_ptr, &list_size);
+        char* const* list = Mat_GetDir(m_pimpl->mat_ptr, &list_size);
 
         outputNames.resize(list_size);
         for (size_t i = 0; i < list_size; ++i)

